exercises/meetup/.meta/hints.md

```diff
@@ -0,0 +1 @@
+In Javascript, the Date object month's index ranges from 0 to 11.
```

Comment:
````suggestion
In Javascript, the Date object month's index ranges from 0 to 11.

```javascript
const date = new Date('2020-06-13')

date.getFullYear()
// => 2020
date.getMonth()
// => 5 (instead of 6)

date.getDate()
// => 13
```
````